compiler/wasm/src/types/noir_artifact.ts

```diff
@@ -145,14 +145,22 @@ export type BrilligFunctionId = number;
 
 export type OpcodeToLocationsMap = Record<OpcodeLocation, SourceCodeLocation[]>;
```

Comment:
Is this type necessary anymore?

Reply:
It is used by brillig_locations, so I guess we need it.